tests/integration_tests/test_applications.py

```diff
@@ -638,6 +638,8 @@
             "10",
             "--data_directory",
             "TESTOUTPUTDIR/",
+            "--output_path",
+            "./",
```

Comment:
Changed this to be the output dir as well. Otherwise it writes to the local directory (sort of).